www_src/main.less

```diff
@@ -56,6 +58,25 @@ code, pre {
   }
 }
 
+// Screen readers only
```

Comment:
little confused by the comment plus actual rules: screen readers only really care about display:none or visibility:hidden, so it seems that we don't need all these rules. A simple "display:none" should already achieve the goal. But then the question becomes: if sr-only is always active for these elements, how does this do different things for screen readers vs. not screen readers?